catalog/USFS/USFS_GTAC_TreeMap_v2016.jsonnet

```diff
@@ -0,0 +1,2351 @@
+    Can. J. For. Res. 39: 1901-1916. [doi:10.1139/X09-102](https://doi.org/10.1139/X09-102).
+
+    * **Riley, Karin L.; Grenfell, Isaac C.; Finney, Mark A.; Shaw, John D. 2021.** 
+    TreeMap 2016: A tree-level model of the forests of the conterminous United States circa 2016. 
+    Fort Collins, CO: Forest Service Research Data Archive. [https://doi.org/10.2737/RDS-2021-0074]https://doi.org/10.2737/RDS-2021-0074.
+
+    * **Wilson, B Tyler, Andrew J Lister, and Rachel I Riemann. 2012.**
+    A Nearest-Neighbor Imputation Approach to Mapping Tree Species over Large Areas Using Forest Inventory Plots and Moderate Resolution Raster Data.
+    Forest Ecol. Manag. 271:182-198. [doi: 10.1016/j. foreco.2012.02.002](https://doi.org/10.1016/j.foreco.2012.02.002).
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license(
+        'https://data.fs.usda.gov/geodata/rastergateway/treemap/'),
+    {
+      rel: ee_const.rel.source,
+      href: 'https://data.fs.usda.gov/geodata/rastergateway/treemap/',
+    },
+    ee.link.latest(latest_id, catalog_subdir_url + latest_basename + '.json'),
+    ee.link.predecessor(
+        predecessor_id, catalog_subdir_url + predecessor_basename + '.json'),
+  ],
+  keywords: [
+    'biota',
+    'environment',
+    'Ecology, Ecosystems, & Environment',
+    'Forest & Plant Health',
+    'Inventory, Monitoring, & Analysis',
+    'Natural Resource Management & Use',
+    'Ecosystem services',
+    'Forest management',
+    'redcastle_resources',
+    'Restoration',
+    'Timber',
+    'Wilderness',
+    'Forest Inventory and Analysis',
+    'FIA',
+    'imputation',
+    'LANDFIRE',
+    'random forests',
+    'tree list',
+    'CONUS',
+    'conterminous United States',
+  ],
+  providers: [
+    ee.producer_provider('USDA Forest Service (USFS) Geospatial Technology and Applications Center (GTAC)', 'https://apps.fs.usda.gov/lcms-viewer/treemap.html'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(-128.97722, 22.76862, -65.25445, 51.64968,
+                    '2016-01-01T00:00:00Z', '2017-01-01T00:00:00Z'),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'year',
+        description: |||
+          Year of the product.
+        |||,
+        type: ee_const.var_type.number,
+      },
+      {
+        name: 'study_area',
+        description: |||
+          TreeMap currently only covers CONUS.
+
+          Possible values: 'CONUS'
+        |||,
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'landfire_ver',
+        description: |||
+          Landfire version used as reference and target data for imputation.
+        |||,
+        type: ee_const.var_type.string,
+      },
+    ],
+    gsd: [30],
+    'eo:bands': [
+      {
+        name: 'ALSTK',
+        description: |||
+          All-Live-Tree Stocking (percent) - The sum of stocking percent values of all live trees on the condition.
```

Comment:
please use units fields instead of putting units in the description: eg, see https://github.com/google/earthengine-catalog/blob/main/catalog/TEMPLATE/TEMPLATE_IMAGE_COLLECTION_V2_3.jsonnet#L181

Reply:
Also, please use the format
"All-Live-Tree Stocking. The sum of stocking percent values of all live trees on the condition."